src/index.ts

```diff
@@ -47,7 +47,7 @@ export {
     AUTO_TX
 } from './table';
 export {getCredentialsFromEnv, getSACredentialsFromJson} from './parse-env-vars';
-export {parseConnectionString, ParsedConnectionString} from './parse-connection-string';
+export {parseConnectionString, IParsedConnectionString} from './parse-connection-string';
```

Comment:
Again, is this renaming worthy of breaking dependencies?

Reply:
As I said [above](https://github.com/ydb-platform/ydb-nodejs-sdk/pull/135#discussion_r810815952), I suppose that impact of these changes is minimal, but it's useful to make interface names consistent.